eslint/babel-eslint-plugin/package.json

```diff
@@ -26,14 +26,14 @@
   },
   "homepage": "https://github.com/babel/babel/tree/master/eslint/babel-eslint-plugin",
   "peerDependencies": {
-    "eslint": ">=4.0.0"
+    "eslint": ">=6.0.0"
   },
   "dependencies": {
     "eslint-rule-composer": "^0.3.0"
   },
   "devDependencies": {
-    "babel-eslint": "^8.2.2",
-    "eslint": "^4.19.1",
+    "babel-eslint": "^10.0.0",
```

Comment:
What's the best way to get this to link `@babel/eslint-parser` here instead since it's not published?